drivers/mrf24j40/Kconfig

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_MODULE_MRF24J40
+    bool "Configure MRF24J40 driver"
+    depends on MODULE_MRF24J40
+    help
+        Configure the MRF24J40 driver using Kconfig.
+
+if KCONFIG_MODULE_MRF24J40
+
+config MRF24J40_USE_EXT_PA_LNA
+    bool "Enable external PN/LNA control"
```

Comment:
Comment should say "PA/LNA" instead of "PN/LNA".

Reply:
Right, fixed!